ddtrace/internal/ci_visibility/telemetry/constants.py

```diff
@@ -0,0 +1,44 @@
+from enum import Enum
+
+
+CIVISIBILITY_TELEMETRY_NAMESPACE = "civisibility"
+
+
+class ERROR_TYPES(Enum):
```

Comment:
```suggestion
class ERROR_TYPES(str, Enum):
```
to save having to access `.value`?